src/components/Layout/Header.js

```diff
@@ -0,0 +1,11 @@
+import logo from '../../imgs/logo.svg';
+
+import StyledHeader from './styledheader.js';
+
+export default function Header() {
+	return (
+		<StyledHeader>
+			<img src={logo} alt="PlatePlan logo with plate and cutlery" />
```

Comment:
I would recommend to just write the text that is on the logo and not describe the logo. That is a bit overkill :)
```suggestion
			<img src={logo} alt="PlatePlan" />
```